developer-docs-site/docs/guides/getting-started.md

```diff
@@ -26,45 +26,52 @@ Start by cloning the `aptos-core` GitHub repo from [GitHub](https://github.com/a
     cd aptos-core
     ```
 
-3. Run the `scripts/dev_setup.sh` Bash script as shown below. This will prepare your developer environment.
+3. Run the `scripts/dev_setup.sh` Bash script as shown below. This will prepare your developer environment by installing all dependencies needed to build, test and inspect Aptos Core. Note, you may be prompted for your password:
```

Comment:
- Has anything changed that this script now prompts for the password? In general none of our scripts require sudo or root privileges and I don't remember seeing such prompts. 
- Also, I hesitate to claim that it installs all the dependencies, because below in this same doc we ask the user to install a few more tools. I had deliberately left it like the above without claiming that it installs all the dependencies.

Reply:
> * Has anything changed that this script now prompts for the password? In general none of our scripts require sudo or root privileges and I don't remember seeing such prompts.
I am not sure if something changed, I get prompted by the script during the `Installing protoc and plugins` part of the flow, as shown below:

```
Pouring node--18.10.0.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/node/18.10.0: 1,993 files, 48.4MB
==> Running `brew cleanup node`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
Removing: /opt/homebrew/Cellar/node/18.9.1... (1,997 files, 48.4MB)
npm is already installed
unzip is already installed
Installing protoc and plugins
Password:
Archive:  protoc-21.4-osx-universal_binary.zip
  inflating: /usr/local/bin/protoc   
Archive:  protoc-21.4-osx-universal_binary.zip
   creating: /usr/local/include/

```

> * Also, I hesitate to claim that it installs all the dependencies, because below in this same doc we ask the user to install a few more tools. I had deliberately left it like the above without claiming that it installs all the dependencies.
I say "most" instead of "all" now. FWIW, the script itself outputs, "This script will download and install the necessary dependencies needed to build, test and inspect Aptos Core."